jupyter/nmm-game-utils.ipynb

```diff
@@ -49,16 +37,32 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Calculates the opponent of the given player\n",
-    "def opponent(p):\n",
-    "    return 'b' if p == 'w' else 'w'"
+    "START_STATE = ((9, 9), (\n",
+    "    (' ', ' ', ' ', ' ', ' ', ' ', ' ', ' '),\n",
+    "    (' ', ' ', ' ', ' ', ' ', ' ', ' ', ' '),\n",
+    "    (' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ')\n",
+    "))"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Stones"
+    "## Funktionen um mit Steinen zu interagieren"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### hasPlaceableStones\n",
+    "\n",
+    "Die Funktion `hasPlaceableStones` überprüft, ob ein Spieler für einen Zustand noch zusetzende Steine auf dem Stapel (_stash_) besitzt. Die Funktion hat zwei Argumente:\n",
+    "\n",
+    "- `s` ist ein Zustand eines Spiels;\n",
+    "- `p` ist ein Spieler.\n",
```

Comment:
Würden wir etwas dadurch gewinnen, dass wir

* $s \in States$ ist ein Zustand;
* $p \in Players$ ist ein Spieler.

schreiben würden? Sähe zumindest wissenschaftlicher aus :D